packages/esm-patient-common-lib/src/dashboards/DashboardExtension.tsx

```diff
@@ -1,15 +1,23 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import { last } from 'lodash';
+
 import { ConfigurableLink } from '@openmrs/esm-framework';
+import styles from './dashboardextension.scss';
 
 export interface DashboardExtensionProps {
   title: string;
   basePath: string;
+  currentPath: string;
 }
 
-export const DashboardExtension = ({ title, basePath }: DashboardExtensionProps) => {
+export const DashboardExtension = ({ title, basePath, currentPath }: DashboardExtensionProps) => {
+  const navLink = useMemo(() => decodeURIComponent(last(currentPath.split('/'))), [currentPath]);
```

Comment:
This would be good to move into its own hook. probably after the react-router v6 upgrade